ui/src/app/base/components/Switch/Switch.tsx

```diff
@@ -0,0 +1,29 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+type Props = {
+  className?: string;
+} & React.PropsWithoutRef<JSX.IntrinsicElements["input"]>;
+
+const Switch = ({ className, ...inputProps }: Props): JSX.Element => {
```

Comment:
Should there be a `label` prop as well? I guess that could wait until we actually need a switch to have a label, actually

Reply:
Yeah I wasn't sure if there should be a base switch and a more advanced one with lables/validation etc.